### PR TITLE
Create security.txt

### PR DESCRIPTION
**Pull request by vcartman**
Created security.txt with the links to the Vulnerability Disclosure Program.

### CODE REVIEW

**Comment by glangendam** [comment]
Attached is a security.txt file that is more in line with RFC 9116. It should be located at divd.nl/.well-known/security.txt

[security.txt](https://github.com/DIVD-NL/sites/files/8734031/security.txt)

File: csirt.divd.nl/_cases/DIVD-2022-00025.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ timeline:
 - start: 2022-04-13
   end:	
   event: First version of this case file.
+- start: 2022-04-13
+  end:
+  event: First batch of notifications sent 
+- start: 2022-05-15
+  end: 
+  event: Second batch of notifications sent
```
Comment (MrSeccubus) [inline csirt.divd.nl/_cases/DIVD-2022-00025.md:38]: Don;t this is this an update of security.txt.

**Review by MrSeccubus — CHANGES_REQUESTED** [review]
Suggestion, lets make this more in line with the RFC.

It should be located in the .well-known directory too and be present on www as well.

**Comment by cookiemonster** [comment]
We have a VDP btw. 
https://www.divd.nl/security/

**Comment by glangendam** [comment]
> @glangendam we deliberately chose the bug bounty progrtam ursl over email. ((And if ywe accept email I would either suggest question@ or csirt@ rather then security@)
 
I added the email because 'some' organizations use automated scanning and emailing of vulnerabilities. They benefit hugely from a standardized location of an email address. I think DIVD should give a good example, maybe?

> Also a number of URL's that you mention in the attachment are not actually working.
My apologies, it was more a suggestion how a security.txt should look according to RFC9116.

> @vcartman / @glangendam who is going to incoraporate these suggetions into the PR?
I have not much experience in this, but I would be happy to.

**Pull request closed without merging**